### ISSUE TITLE
 hyperparameter choices 

### ISSUE THREAD

**Issue by hithd** [GaoShuang98/CVCities#10]
Could the authors provide the specific batch size, learning rate, and other hyperparameter choices for CVUSA, CVACT, and VIGOR? The results are not satisfactory when directly using the learning rate specified in the code.

**Comment by GaoShuang98** [comment]
The train_**.py files indicates some training examples where the parameters are not optimal. Now the train_**.py files has been updated.

**Comment by hithd** [comment]
Thank you, it works well now.
